oio/xcute/common/job.py

```diff
@@ -58,13 +71,7 @@ def load_config(self, job_config):
     def get_tasks(self, marker=None):
         """
             Yields the job tasks as
-            (TaskClass, task_id, task_payload, total_tasks)
+            (task_id, task_payload, total_tasks)
```

Comment:
In which case would a job want to send different tasks? And even if it were, it would be enough to send the information to the payload so that the task processing uses the correct code.

Reply:
So the only reason we were sending the task at the beginning was because we wanted to pickle it? Ok